packages/nimble-components/src/select/styles.ts

```diff
@@ -116,6 +118,18 @@ export const styles = css`
         overflow: auto;
     }
 
+    ::slotted([role='option'][active-option]) {
+        background-color: ${fillSelectedColor};
+    }
+
+    ::slotted([role='option']:not([active-option])) {
+        background-color: none;
```

Comment:
This styling isn't working because `none` isn't a valid value for `background-color`. You could either do `background: none` or `background-color: transparent`.

It would be nice to have a matrix test to catch things like this. Could you write a test case where the select is open and has a different active option than the selected option?

Reply:
Added a matrix story that navigates an open dropdown. It indeed would have caught that styling goof.